monitoring/prometheus/ymls/load-tests.yml

```diff
@@ -0,0 +1,18 @@
+  ####################
+  # monitor load tests
+  ####################
+
+  # monitor a load test container that is colocated on prometheus-grafana-metrics
+
+  # disabled since the test was concluded successfully
```

Comment:
I'm impartial to removing this.

Reply:
yeh, don't love putting it here but also don't know where else to put it
fine to leave as is imo